test/com/dotmarketing/plugin/PluginMergerTest.java

```diff
@@ -8,12 +8,20 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import com.dotcms.TestBase;
 import com.dotmarketing.plugin.util.PluginFileMerger;
+import com.dotmarketing.util.IntegrationTestInitService;
 
 public class PluginMergerTest extends TestBase {
+	
+	@BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
```

Comment:
This is supposed to be unit, please remove the @BeforeClass and remove the extends TestBase (not needed and makes it fail). Also needs to be moved to unit-test package